lnwire/lnwire.go

```diff
@@ -171,12 +171,35 @@ func writeElement(w io.Writer, element interface{}) error {
 			}
 		}
 	case *btcec.Signature:
+		// Serialize the signature with all the checks that entails
 		sig := e.Serialize()
-		if len(sig) > 73 {
-			return fmt.Errorf("Signature too long!")
+		var b [64]byte
+		// Extract lengths of R and S
```

Comment:
Minor nit, but can you add a comment here which explains that we grab from an index of `3` since the first three bytes within the `DER` encoding are `0x03 <length> 0x02`, with the length of `R` being the `4th byte` and similarly a comment which explains that the length of `R` there's an additional `0x02` byte which needs to be skipped. 

Referring to the godoc for the [`Serialize`](https://godoc.org/github.com/btcsuite/btcd/btcec#Signature.Serialize) method might be helpful.